app/views/notes/_responses.html.erb

```diff
@@ -5,7 +5,15 @@
     <% @responses = DrupalTag.find_nodes_by_type("#{response_type}:#{@node.id}", "note", 200) %>
     <% @unpaginated = true %>
  
-    <p id="<%= response_type %>s"><a class='btn btn-primary btn-lg' href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0 %>"><%= t('notes._responses.post_' + response_type) %></a></p>
+    <p id="<%= response_type %>s"><a class='btn btn-primary btn-lg' href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0 %>"><%= t('notes._responses.post_' + response_type) %></a>&nbsp;
+      <% if @node.power_tag('activity') %>
+        <% if DrupalTag.where(name: @node.power_tag('activity')).first %>
+          <% if DrupalTag.where(name: @node.power_tag('activity')).first.nodes %>
+            <%= DrupalTag.where(name: @node.power_tag('activity')).first.nodes.first.path %>
```

Comment:
Not sure if this answers your question, but here, we now can make this appear in a link, like:

```ruby
<a href="<%= DrupalTag.where(name: @node.power_tag('activity')).first.nodes.first.path %>">Browse other activities for "<%= @node.power_tag('activity') %>"</a>
```

Make sense?